invenio_app_ils/documents/mappings/os-v1/documents/document-v2.0.0.json

```diff
@@ -145,12 +171,17 @@
               },
               "full_words": {
                 "type": "text",
-                "analyzer": "min_length_2",
-                "search_analyzer": "min_length_2"
+                "analyzer": "custom_analyzer",
+                "search_analyzer": "custom_analyzer"
               },
               "normalized_keyword": {
                 "type": "keyword",
                 "normalizer": "custom_normalizer"
+              },
+              "min_length_2": {
```

Comment:
I am not fully understanding why would you add this as a field, could you explain what is the difference?

Reply:
I applied the analyzer as well to this field to be used by the cross_fields type, while at the same time keeping the behaviour of before by keeping the min_length_2 (which will index tokens of at least 2 characters). 

Alternatively we could add the min_length_2 in the `custom_analyzer` but it would imply that all the rest of the fields using this would also start indexing tokens of at least 2, which might not be needed for many of them

Reply:
it won't work well for authors names, because some names are 2 char long, my hesitation is that I find it a bit strange to have a dedicated field for this, since in the other fields by default we will have more tokens (not only the ones longer than 2 chars)